controllers/auth.js

```diff
@@ -1,23 +1,70 @@
+const passport = require("passport");
+const User = require("../models/user")
+  
 var getLogin = (req, res) => {
   //TODO: render login page
+  res.render("login", { title: "Login" });
 };
 
 var postLogin = (req, res) => {
```

Comment:
This function for logging in does not seem to work. Even with a wrong password I can log into my account

Reply:
okay i'll look into this and try to fix this issue.